protocols/kad/src/behaviour.rs

```diff
@@ -867,6 +894,9 @@ where
     ///
     /// > **Note**: Bootstrapping requires at least one node of the DHT to be known.
     /// > See [`Behaviour::add_address`].
+    /// > **Note**: The bootstrapping interval is used to call bootstrap periodically
+    /// to ensure a healthy routing table.
+    /// > See bootstrap_interval field in Config.
```

Comment:
I'd like to mention this more prominently. Ideally, the first sentence after the heading of these docs says something like: "Typically, you don't need to call this function yourself." and then explain the configuration and what it does.

Reply:
Added some more details regarding that